mxd/3.2_MXD_tutorial2_Access_policies.md

```diff
@@ -0,0 +1,129 @@
+# Restricting users from accessing an asset
+
+Bob will once again be the data provider and Alice is interested in Bob’s data assets. Bob, as a data provider, creates an asset. The data asset should have the following properties:
+
+| ID          | 4                                                                                            |
+|-------------|----------------------------------------------------------------------------------------------|
+| Description | Product EDC Demo Asset 4                                                                     |
+| Type        | HttpData                                                                                     |
+| URL         | [https://jsonplaceholder.typicode.com/todos/4](https://jsonplaceholder.typicode.com/todos/4) |
+
+Action (Bob): Create an asset using the following curl command:
+
+```shell
+curl --location 'http://localhost/bob/management/v2/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "asset": {
+        "@type": "Asset",
+        "@id": "4", 
+        "properties": {
+            "description": "Product EDC Demo Asset 4"
+        }
+    },
+    "dataAddress": {
+        "@type": "DataAddress",
+        "type": "HttpData",
+        "baseUrl": "https://jsonplaceholder.typicode.com/todos/4"
+    }
+}'
+```
+
+Now that the asset is created, a policy must be created to define who can access the asset. This time Bob does not want Alice to see and access the asset. So he defines a policy not allowing Alice to access his asset.
```

Comment:
please be more specific regarding "access" to the asset. we are defining an `accessPolicy`, which restricts the contents of the catalog. "Access" could also lead people to believe we mean "allowed to transfer"